languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35382,7 +35382,50 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
-    <rulegroup id="POUR_PAS_QUE" name="familier : pour (ne) pas que...">
+      <rulegroup id="JE_M_EXCUSE " name="Je m’excuse" type="style" default="temp_off">
+      <rule>
+          <pattern>
+              <token postag="SENT_START|M nonfin" postag_regexp="yes"/>
+              <marker>
+                  <token>je</token>
+                  <token>m'</token>
+                  <token>excuse</token>
+              </marker>
+              <token postag="P.*|M fin" postag_regexp="yes"/>
+          </pattern>
+          <message>S'excuser soi-même peut être perçu comme un manque de politesse. Souhaitez-vous écrire <suggestion>excusez-moi</suggestion> employé dans le langage courant ou <suggestion>veuillez m'excuser</suggestion>
+              employé dans le langage formel ou <suggestion>je vous prie de m'excuser</suggestion> employé dans le langage soutenu ?</message>
+          <url>https://fr.wiktionary.org/wiki/je_m%E2%80%99excuse</url>
+          <example correction="Excusez-moi|Veuillez m'excuser|Je vous prie de m'excuser"><marker>Je m'excuse</marker> de vous déranger.</example>
+          <example correction="excusez-moi|veuillez m'excuser|je vous prie de m'excuser">Je ne souhaitais pas cela, <marker>je m'excuse</marker>.</example>
+      </rule>
+          <rule>
+      <pattern>
+          <token regexp="yes" skip="-1">tu|te</token>
```

Comment:
Why do we need this token?
I don't see an example to illustrate why it's needed.